### PR TITLE
[WIP] Distributed oCIS

### PR DESCRIPTION
**Pull request by IljaN**
- [ ] Add document start marker
- [ ] Make sharing work (shares.json could not be created on boot)
- [ ] Make this work without /etc/hosts entry to ocis-proxy on the host

### CODE REVIEW

File: examples/distributed-ocis/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,158 @@
+version: '3.7'
```
Comment (xoxys) [inline examples/distributed-ocis/docker-compose.yml:1]: Just add the document start marker `---` as you did in the other YAML file :) and maybe you want to clean up some empty lines at the end of the file. Beside that LGTM

**Review by refs — APPROVED** [review]
removing commented bits? LGTM!

File: examples/distributed-ocis/docker-compose.yml (hunk 1 above)
Comment (michaelstingl) [inline examples/distributed-ocis/docker-compose.yml:1]:
```suggestion
---
version: '3.7'
```

**Comment by felixboehm** [comment]
fixes in follow up https://github.com/owncloud-docker/compose-playground/issues/18

**Pull request merged**